releases/client/nightly/settings

```diff
@@ -0,0 +1 @@
+FOREMAN_VERSION='nightly'
```

Comment:
We set this by default:
https://github.com/theforeman/theforeman-rel-eng/blob/6a9a8a39e3b0a4889102c7add4132a624a1ef5cb/releases/client/settings#L3
```suggestion
```